techniques/css/C42.html

```diff
@@ -18,67 +18,50 @@ <h2>When to Use</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to ensure that links in navigation or pagination menus will be spaced so that they fall within an area that measures at least 44 x 44 CSS pixels if the target area itself is smaller than that. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>	
-		
+			<p>The objective of this technique is to ensure that navigation or pagination links will be sized so that they fall within an area that measures at least 44 &#215; 44 CSS pixels. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>
```

Comment:
as said before:

"ensure that navigation or pagination links measure at least 44 by 44 CSS pixels." and remove the rest of the sentence.

Reply:
That's already there—I'm not sure why you're seeing the old sentence, but that's not what I'm seeing. Here's a screen grab of what I see:
<img width="823" alt="screen grab of the relevant text from the raw git page" src="https://github.com/w3c/wcag/assets/20049493/a14a5594-228e-4f56-a9aa-53facd01a63f">

Reply:
well exactly. that whole thing should just say:

"The objective of this technique is to ensure that navigation or pagination links measure at least 44 by 44 CSS pixels." and remove the rest of the sentence."

and then nothing else. Nothing about "adequate target clearance" or "offset", or "accidental pointer activation". Just the above sentence.

Reply:
At a stretch, if we want to have an aim:

"The objective of this technique is to ensure that navigation or pagination links measure at least 44 by 44 CSS pixels." and remove the rest of the sentence. The aim is to that target sizes are large enough for users to easily activate them."

Reply:
> and remove the rest of the sentence.
> and then nothing else. Nothing about "adequate target clearance" or "offset", or "accidental pointer activation". Just the above sentence.

Ah. You mean the rest of the sentence after "44 CSS pixels" and then the entire second sentence / rest of the paragraph.